infra/aws/main.tf

```diff
@@ -0,0 +1,163 @@
+resource "buildkite_agent_token" "tf_managed" {
```

Comment:
could you add a readme file on how to init and use these terraform files? like what kind of aws credential it should use? where are the states being saved etc.?

Reply:
done